lib/resources/may2015/medication_statement.yaml

```diff
@@ -0,0 +1,282 @@
+                    "id": "2750067",
+                    "code": {
+                      "coding": [
+                        {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+                         "code": "1191",
+                         "display": "Aspirin",
+                         "userSelected": false}],
+                      "text": "aspirin"}}]}
+  url: http://hl7.org/fhir/2015May/domainresource-definitions.html#DomainResource.contained
+  note: It should contain medication resource with medication.code
+  children:
+
+  - name: code
+    required: 'No'
+    cardinality: 0..1
+    type: CodeableConcept
+    description:  Codes that identify this medication.
+    example: |
+               {"code": {
+                 "coding": [
+                   {"system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+                    "code": "1191",
+                    "display": "Aspirin",
+                    "userSelected": false}],
+                 "text": "aspirin"}}
+    url: http://hl7.org/fhir/2015May/medication-definitions.html#Medication.code
+    note: This code is only belongs where the resource is a Medication.
+    binding:
+      description: A code that defines the type of medication.
+      terminology:
+      - display: RxNorm
+        system: http://www.nlm.nih.gov/research/umls/rxnorm
+
+- name: patient
+  required: 'Yes'
+  cardinality: 0..1
+  type: Reference (Patient)
+  description: Who is/was taking the medication.
+  example: '{"patient": {"reference": "Patient/5366327"}}'
+
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: >
+                 A code representing the patient or other source's judgment about the state of the medication
+                 used that this statement is about.
+  example: '{"status": "in-progress"}'
+
+- name: effectivePeriod
+  required: 'No'
+  cardinality: 0..1
+  type: Period
+  description: The interval of time during which it is being asserted that the patient was taking the medication.
+  note: If effectivePeriod and dosage.timing.repeat.boundsPeriod are both populated they need to be the same value.
+  example: |
+             {"effectivePeriod": {
+                "start": "2015-05-05T14:00:00.000Z",
+                "end": "2015-05-06T01:00:00.000Z"}}
+  url: http://hl7.org/fhir/2015May/medicationstatement-definitions.html#MedicationStatement.effective_x_
+  action: create
+
+- name: note
+  required: 'No'
+  cardinality: 0..1
+  type: string
+  description: Provides extra information about the medication statement that is not conveyed by the other attributes.
+  example: '{"note": "do not take with alcohol"}'
+  action: create
+
+- name: medication
+  required: 'Yes'
+  cardinality: 0..1
+  type: contained Reference (Medication)
+  description: Identifies the medication being administered.
+  note:
+  - update: This must be the current medicationReference.code. It cannot be modified.
```

Comment:
How does limiting this medication field to create affect this update specific note? Can it be removed?